### DISCUSSION TITLE
When Gallery's all editing features are added?

### DISCUSSION THREAD

**Discussion by iBabu007** [FossifyOrg/General-Discussion#399] [General]
when all editing features are added to Fossify gallery. It is necessary part of gallery and it is difficult to use it without these features.
Please add it as soon as possible.

**Comment by Aga-C** [comment]
Currently, we don't have any ETA for advanced editor. For updates and more info please use this discussion: https://github.com/FossifyOrg/Gallery/discussions/90.
